### PR TITLE
Avoid overlapping brackets

### PR DESCRIPTION
**Pull request by mike-spa**
Resolves: #15456

### CODE REVIEW

File: src/engraving/libmscore/edit.cpp [hunk 1]
```diff
@@ -6027,6 +6027,17 @@ void Score::undoChangeTpc(Note* note, int v)
 
 void Score::undoAddBracket(Staff* staff, int level, BracketType type, size_t span)
 {
+    staff_idx_t startStaffIdx = staff->idx();
+    staff_idx_t totStaves = nstaves();
+    // Make sure this brackets won't overlap with others sharing same column.
```
Comment (RobFog) [inline src/engraving/libmscore/edit.cpp:6032]: "this" → "that"

**Review by oktophonie — APPROVED** [review]

File: src/engraving/libmscore/edit.cpp [hunk 2]
```diff
@@ -6027,6 +6027,17 @@ void Score::undoChangeTpc(Note* note, int v)
...
+    // If overlaps are found, move the other brackets outwards (i.e. increase column).
+    for (staff_idx_t staffIdx = startStaffIdx; staffIdx < startStaffIdx + span && staffIdx < totStaves; ++staffIdx) {
+        for (BracketItem* bracketItem : staves().at(staffIdx)->brackets()) {
```
Comment (RomanPudashkin) [inline src/engraving/libmscore/edit.cpp:6035]: You should use _staves directly here

**Comment by RomanPudashkin** [comment]
@mike-spa please rebase it

**Comment by RomanPudashkin** [comment]
@mike-spa any update on this? :)

> @mike-spa please rebase it

**Comment by mike-spa** [comment]
Oops, lost sight of this one! Done and rebased 👍

File: src/engraving/libmscore/edit.cpp (hunk 1 above)
Comment (RobFog) [reply src/engraving/libmscore/edit.cpp:6165]: This should read "that", not "this", right?

**Pull request merged**